bionic.Dockerfile

```diff
@@ -0,0 +1,242 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2020 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM ubuntu:bionic
+
+RUN \
+    apt-get -q update \
+    && apt-get install -y \
+       openssh-server \
```

Comment:
@Nirothipan minor change. `openssh-server` is not in order.

Reply:
Thanks . Fixed it .